forms-flow-api/migrations/versions/0e526ddca7b3_theme.py

```diff
@@ -0,0 +1,40 @@
+"""theme
+
+Revision ID: 0e526ddca7b3
+Revises: fdfe787a197c
+Create Date: 2024-04-22 01:44:13.085411
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0e526ddca7b3'
+down_revision = 'fdfe787a197c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('theme_customization',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('logo_name', sa.String(length=50), nullable=False),
+    sa.Column('logo_type', sa.String(length=100), nullable=False),
+    sa.Column('value', sa.String(length=20), nullable=False),
```

Comment:
Is the value intended for logo ? Please rename it to make it more clear.
Also value can be base64 string, so length 20 wouldn't work. Make it flexible by not adding length.